File: Tests/installation_tests/cocoapods/CocoapodsTest/ViewController.m
//
//  ViewController.m
//  CocoapodsTest
//
//  Created by Jack Flintermann on 5/7/15.
//  Copyright (c) 2015 stripe. All rights reserved.
//

#import "ViewController.h"
#import <Stripe/Stripe.h>

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [Stripe setDefaultPublishableKey:@"test"];
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: Tests/installation_tests/cocoapods/with_frameworks/CocoapodsTest/ViewController.swift
//
//  ViewController.swift
//  CocoapodsTest
//
//  Created by Jack Flintermann on 8/4/15.
//  Copyright (c) 2015 jflinter. All rights reserved.
//

import UIKit
import Stripe

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        Stripe.setDefaultPublishableKey("test")
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

